scripts/integration-test-pipelines.sh

```diff
@@ -9,24 +9,44 @@ $("${SCRIPTS_DIR}/environment.sh")
 "${SCRIPTS_DIR}/fly_sync_and_login.sh"
 
 generate_vars_file() {
-   cat <<EOF
+SSH_KEY=$(aws s3 cp s3://"${STATE_BUCKET_NAME:-gds-paas-${DEPLOY_ENV}-state}"/ci_build_tag_key -)
+  cat << EOF
 ---
+pipeline_name: ${pipeline_name}
 github_access_token: ${GITHUB_ACCESS_TOKEN}
 github_status_context: ${DEPLOY_ENV}/status
-github_repo: ${github_repo}
+state_bucket: ${STATE_BUCKET_NAME:-gds-paas-${DEPLOY_ENV}-state}
+aws_region: ${AWS_DEFAULT_REGION:-eu-west-1}
+branch_name: ${BRANCH:-master}
+organisation: ${organisation}
+repository: ${repository}
+github_repo: ${organisation}/${repository}
+github_repo_uri: git@github.com:${organisation}/${repository}.git
+tag_branch: ${tag_branch}
+version_file: ${version_file}
+secrets_file: ${secrets_file}
 EOF
+echo -e "tagging_key: |\n  ${SSH_KEY//$'\n'/$'\n'  }"
 }
 
 setup_test_pipeline() {
   pipeline_name="$1"
-  github_repo="$2"
+  version_file="version"
+  organisation="$2"
+  repository="$3"
+  tag_branch="$4"
+  secrets_file="${5:-no-secrets-needed}"
```

Comment:
Added in https://github.com/alphagov/paas-release-ci/pull/30/files#diff-203673b1eaa54b56dcea63cd041ec574R119